c7n/resources/resource_map.py

```diff
@@ -165,6 +166,7 @@
   "aws.message-broker": "c7n.resources.mq.MessageBroker",
   "aws.message-config": "c7n.resources.mq.MessageConfig",
   "aws.mirror-session": "c7n.resources.vpc.TrafficMirrorSession",
+  "aws.model-customization-job": "c7n.resources.bedrock.BedrockModelCustomizationJob",
```

Comment:
hmm.. would like to keep this prefixed with bedrock, there's a few model / job services in aws already. maybe `bedrock-customization-job` ?

Reply:
Makes sense to me. Addressed